src/QuadForm/Torsion.jl

```diff
@@ -70,22 +74,24 @@ function torsion_quadratic_module(M::ZZLat, N::ZZLat; gens::Union{Nothing, Vecto
   end
   # mS : S -> A
   # generators of S lifted along M -> M/N = A -> S
-  if gens !== nothing  && length(gens)>0
+  if gens !== nothing  && length(gens) > 0
     gens_lift = gens
   else
     gens_lift = Vector{QQFieldElem}[reshape(collect(change_base_ring(FlintQQ, mS(s).coeff) * BM), :) for s in Hecke.gens(S)]
   end
 
   num = basis_matrix(M) * gram_matrix(ambient_space(M)) * transpose(basis_matrix(N))
   if iszero(modulus)
-    modulus = reduce(gcd, num; init = zero(QQFieldElem))
+    _modulus = reduce(gcd, num; init = zero(QQFieldElem))
```

Comment:
why all these underscores?

Reply:
If the input is `1//1`, then `modulus` would change it's type here.